app/controllers/concerns/include_concern.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Concerns
+  module IncludeConcern
+    extend ActiveSupport::Concern
+
+    def permitted_include
+      return '' unless permitted_inclusions && include_params[:include]
```

Comment:
Shouldn't this be `[]` instead of `''`?

Reply:
Can be any of both. https://github.com/rails-api/active_model_serializers/blob/0-10-stable/docs/general/adapters.md#include-option